perf/stability/control-plane/test.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup() {
+  kubectl create ns twopods-istio
+  kubectl create ns invalid-namespace-0
+  kubectl apply -f ../../istio-install/tmp/istio-1.12.0/samples/httpbin/httpbin.yaml -n twopods-istio
```

Comment:
this hard-codes the istio version `istio-1.12.0` in the path, isn't it going to fail after the upgrade?

Reply:
it could be changed, should I just create my own samples where I can put all these required yaml

Reply:
If you do not need to change the yaml, you could just apply from the github like `https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml`

Reply:
so istioctl is not able to locate files which are outside of the code directory, hence I added a samples directory. I might want to change the yaml as well while testing it further for multiple workloads.

Reply:
It does not look right to hard-code the istio version in the path, you should use some env variables, something like this https://github.com/istio/tools/blob/master/perf/stability/gateway-bouncer/setup.sh#L26

Reply:
oh I have added `samples (httpsbin and sleep)` in the same directory. And this line is not necessary as I am deploying the same resources in the next two lines. Will update the code. Thanks